### ISSUE TITLE
Refactor profile form and view to use custom forms

### ISSUE THREAD

**Issue by uoodsq** [rubyforgood/pet-rescue#670]
Post #628, need to refactor the view for viewing and editing profiles to render using custom forms instead of the static profile models.

**Comment by kasugaijin** [comment]
@uoodsq shall I assign you to this and #671 ?

**Comment by uoodsq** [comment]
Yes, thanks for the nudge, will try to get to these in the next week or two. 😀

**Comment by kasugaijin** [comment]
Awesome! Thank you. This will be such a great feature for the app.

**Comment by kasugaijin** [comment]
Hey @uoodsq are you still planning to do these? :)
